datalad_fuse/fuse_.py

```diff
@@ -92,18 +95,39 @@ def _filter_stat(st):
     def getattr(self, path, fh=None):
         # TODO: support of unlocked files... but at what cost?
         lgr.debug("getattr(path=%r, fh=%r)", path, fh)
+        r = None
         if fh and fh < self._counter_offset:
             lgr.debug("Calling os.fstat()")
             r = os.fstat(fh)
         elif op.exists(path):
             lgr.debug("File exists; calling os.stat()")
             r = self._filter_stat(os.stat(path))
+        elif op.lexists(path):
```

Comment:
@yarikoptic Just to be clear: Due to the `op.exists(path)` above, this block is entered iff `path` is a broken symlink.  Is that your intention?

Reply:
yes, my idea was that it is the condition for an annex'ed file with content not locally available.